### PR TITLE
Grammar and reference fixes

### PR DESCRIPTION
**Pull request by goldim**
Done other parts of docs except desktop which was done in prev PR

### CODE REVIEW

**Review by cboulanger — APPROVED** [review]
Again, thanks for doing this!

File: docs/development/howto/icon_fonts.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ let atom = new qx.ui.basic.Atom("Look, I'm a font icon", "@FontAwesome/f004");
 ## Drawbacks
 
 Only `qx.ui.basic.Image` and `qx.ui.table.cellrenderer.Image` support icon
-fonts. It is not possible to use these icon font based images in decorators.
+fonts. It is not possible to use these icon fonts based images in decorators.
```
Comment (derrell) [inline docs/development/howto/icon_fonts.md:140]: s/icon fonts based/icon fonts-based/

File: docs/development/howto/internationalization.md [hunk 2]
```diff
@@ -124,7 +123,7 @@ If the string given to `tr` does not have a translation under the current
 locale, the string itself will be returned.
 
 If the string given to `tr` is the empty string, the header of the .po file is
-returned (which can be a bit confusing if done accidentally, but is correct
+returned which can be a bit confusing if done accidentally, but is correct
```
Comment (derrell) [inline docs/development/howto/internationalization.md:126]: s/returned which/returned, which/
(add comma)

File: docs/mobile/mobile_overview.md [hunk 3]
```diff
@@ -128,7 +124,7 @@ You can contribute in different ways:
 
 - Theming: You can optimize the current CSS files or even create your own theme.
 
-- Widgets: Widget / Feature missing? Create an widget an post it back to us.
+- Widgets: Widget / Feature missing? Create a widget a post it back to us.
```
Comment (derrell) [inline docs/mobile/mobile_overview.md:127]: s/Create a widget a post/Create a widget and submit it as a Pull Request/

(I think that's what's intended here...???)

File: docs/release-notes.6.0.md [hunk 4]
```diff
@@ -118,7 +118,10 @@ In hindsight of better keyboard accessibility support some components received c
 
 ## Accessibility
 
-Accessibility is an important topic in nowadays webdevelopment in order to allow assistive technologies to better help persons with disabilities. Qooxdoo now supports better integration of [WAI-ARIA](https://www.w3.org/TR/wai-aria-1.1/) and more keyboard accessibility for the following components:
+Accessibility is an important topic in nowadays web development
```
Comment (derrell) [inline docs/release-notes.6.0.md:121]: s/nowadays/modern/

File: docs/tutorial/twitter/tutorial-part-2.md [hunk 5]
```diff
@@ -85,38 +85,38 @@ flexible. The second row does the same for the first column.
 
 The last thing we need to fix was the invisible toolbar. If you know the reason
 why it's not visible, you sure know how to fix it. It contains not a single
-element so it won't be visible. Fixing it means adding an element, in our case
+element, so it won't be visible. Fixing it means adding an element, in our case
```
Comment (derrell) [inline docs/tutorial/twitter/tutorial-part-2.md:88]: s/It contains not a single elelment/It doesn't contain any elements/

File: docs/tutorial/twitter/tutorial-part-2.md [hunk 6]
```diff
@@ -179,11 +179,11 @@ reloadButton.addListener(
 ```
 
 Here we see two things: First, how to add an event listener and second, that
-firing an event is as easy as a method call. The only parameter to .fireEvent()
+firing an event is as easy as a method call. The only parameter to `.fireEvent()`
 is the name of the event we have declared in the class definition. Another
 interesting thing here is the third parameter of the `addListener` call,
 **this**. It sets the context of the callback function to our window instance,
-so the this in this.fireEvent() is resolved correctly.
+so this in this.fireEvent() is resolved correctly.
```
Comment (derrell) [inline docs/tutorial/twitter/tutorial-part-2.md:186]: s/this/`this`/

File: docs/tutorial/twitter/tutorial-part-2.md [hunk 7]
```diff
@@ -258,7 +258,7 @@ we need to run the compiler once more. After that, the image should be in the
 windows caption bar.
 
 Two more minor things are left to finish. First, the button does not look very
-good. Why don't we just give it a fixed width to fit its height.
+good. Why don't we just give it a fixed width to fit its height?
```
Comment (derrell) [inline docs/tutorial/twitter/tutorial-part-2.md:261]: This is a rhetorical question, not really a question. I'd either leave it as a period, or replace the period with a colon (`:`)

**Review by derrell — CHANGES_REQUESTED** [review]

**Comment by goldim** [comment]
Big thanks @derrell gonna fix it soon

File: docs/core/data_binding/controller.md [hunk 8]
```diff
@@ -202,38 +201,37 @@ put the corresponding Node object into the selection array of the controller.
 
 ## Form Controller
 
-Also forms do have a special controller. The form controller uses a
-`qx.ui.form.Form` as target and a
-`Object controller data_binding/controller.md#object_controller` for the
+Also, forms do have a special controller. The form controller uses a
```
[the remark this answers is not in the document]
Comment (goldim) [reply docs/core/data_binding/controller.md:204]: Heard that do or does could emphasize a verb in affirmative  sentences. Maybe is it this case?

[the remark this answers is not in the document]
Comment (derrell) [reply docs/core/data_binding/controller.md]: It's an extreme emphasis, e.g., "You DO want to go" is much stronger than just, "You want to go." In this case the extreme emphasis isn't appropriate.

[the remark this answers is not in the document]
Comment (goldim) [reply docs/core/data_binding/controller.md]: Got it, thanks for clarifying

**Review by hkollmann — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Review by derrell — APPROVED** [review]

**Pull request merged**